hadoop-hdds/docs/content/design/ozone-volume-management.md

```diff
@@ -4,7 +4,7 @@ summary: A simplified version of mapping between S3 buckets and Ozone volume/buc
 date: 2020-04-02
 jira: HDDS-3331
 status: accepted
-author: Marton Elek, Arpit Agarwall, Sunjay Radia
+author: Marton Elek, Arpit Agarwal, Sanjay Radia
```

Comment:
You can remove my name, I really cannot claim any authorship credit for this idea. 🙂

Reply:
@arp7 You wrote additional section to this document about the volume ownership problems.